browser/ui/browser_commands.cc

```diff
@@ -898,6 +903,45 @@ void ScrollTabToBottom(Browser* browser) {
   contents->ScrollToBottomOfDocument();
 }
 
+namespace {
+
+class BookmarksExportListener : public ui::SelectFileDialog::Listener {
```

Comment:
Could you add a comment saying that the lifetime of this class is tied to a FileSelected dialog, and that it will be automatically deleted when the dialog is closed/selected/cancelled.